src/main/resources/interface-specification.yml

```diff
@@ -481,6 +529,19 @@ paths:
                 $ref: '#/components/schemas/Problem'
 components:
   schemas:
+    Client:
+      type: object
+      additionalProperties: false
+      properties:
```

Comment:
not sure if more details need to be exposed (keeping in mind that also the producer could use the endpoint)

Reply:
I expect that the producer will be the most frequent user of this endpoint.
For now, these details are enough.